### ISSUE TITLE
Change bulk upload response format

### ISSUE THREAD

**Issue by DenTray** [RonasIT/laravel-media#42]
Need to remove useless `data` wrapper for the bulk upload API

<img width="1104" alt="image" src="https://github.com/user-attachments/assets/725cd56e-e161-4fbd-b366-f616b4c64d57" />

**Comment by AZabolotnikov** [comment]
@DenTray I check it. in tests i ca't saw this case for bulk create in master branch

**Comment by DenTray** [comment]
@AZabolotnikov yep, seems it already fixed but task is not updated, unfortunately, I can't release new version cause the current DEV is broken and need to fix it here before https://github.com/RonasIT/laravel-media/issues/47

**Comment by DenTray** [comment]
Released in https://github.com/RonasIT/laravel-media/releases/tag/0.1
